src/components/themed/WalletList.js

```diff
@@ -161,39 +160,30 @@ export function WalletList(props: Props) {
           key: walletId
         })
       } else if (wallet != null) {
-        const { enabledTokens, name, currencyCode, currencyNames } = asSafeDefaultGuiWallet(wallet)
+        const currencyConfig = account.currencyConfig[wallet.pluginId]
+        const { builtinTokens = {}, currencyInfo, customTokens = {} } = currencyConfig
```

Comment:
We actually need to subscribe to the custom tokens, since those are subject to change. We can probably use the `useWalletSubscriptions` hook introduced in #3295 to accomplish this in a simple & dynamic way.

Reply:
This is fixed now. I did not actually need `useWalletSubscriptions`, since the list of plugins is static. I will need `useWalletSubscriptions` later when we start tracking the enabled tokens as well.